spec/features/category_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'Category', type: :feature do
```

Comment:
- [ ] You made great effort on this project. However, you have some failing tests. Kindly fix them to make your project approve.

<img width="1143" alt="Screen Shot 2022-10-14 at 21 22 49" src="https://user-images.githubusercontent.com/69011963/195895072-7f852d2d-ae73-4d21-a1a1-28af1431c72f.png">